.github/workflows/test_terraform_module.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+# Attention: valid for K8s charms only
+
+name: Terraform module tests
+
+on:
+  pull_request:
+    paths:
+      - 'terraform/**'
+
+permissions:
+  contents: read
+
+jobs:
+  test-terraform:
```

Comment:
Do you think it makes sense to this workflows as a new workflow on the [operator workflows](https://github.com/canonical/operator-workflows)?

Reply:
TBH I was considering removing this one from the template since is specific to k8s, depends on the charm being published in charmhub etc. wdyt

Reply:
good point, I added a condition for preventing the workflow to run with a comment